Assets/RGLUnityPlugin/Scripts/RadarModels/RadarConfiguration.cs

```diff
@@ -13,35 +13,54 @@
 // limitations under the License.
 
 using System;
+using System.Linq;
 using UnityEngine;
 
 namespace RGLUnityPlugin
 {
     [Serializable]
-    public class RadarConfiguration
+    public struct RadarParametersScope
     {
-        [Min(0.0f)] public float minRange;
-
-        [Min(0.0f)] public float maxRange;
+        [Tooltip("Beginning distance range for the parameters.")]
+        [Min(0.0f)] public float beginDistance;
+        [Tooltip("End range distance for the parameters")]
+        [Min(0.0f)] public float endDistance;
+        [Tooltip("Maximum distance difference to create a new radar detection")]
+        [Min(0.0f)] public float distanceSeparationThreshold;
+        [Tooltip("Maximum radial speed difference to create a new radar detection")]
+        [Min(0.0f)] public float radialSpeedSeparationThreshold;
+        [Tooltip("Maximum azimuth difference to create a new radar detection")]
+        [Min(0.0f)] public float azimuthSeparationThreshold;
+    }
 
+    [Serializable]
+    public class RadarConfiguration
+    {
+        [Tooltip("Minimum azimuth angle (left)")]
         [Range(-180.0f, 180.0f)] public float minAzimuthAngle;
 
+        [Tooltip("Maximum azimuth angle (right)")]
         [Range(-180.0f, 180.0f)] public float maxAzimuthAngle;
 
+        [Tooltip("Minimum elevation angle (down)")]
         [Range(-180.0f, 180.0f)] public float minElevationAngle;
 
+        [Tooltip("Maximum elevation angle (up)")]
         [Range(-180.0f, 180.0f)] public float maxElevationAngle;
 
-        [Min(0.0f)] public float rangeSeparation;
-        [Min(0.0f)] public float azimuthSeparation;
+        [Tooltip("Radar scopes of parameters")]
+        public RadarParametersScope[] parametersScopes;
```

Comment:
I would not call this "scope of parameters". Aren't these rather scope parameters? In other words, radar scope is defined by its range and separation, that can be called /radar/ scope parameters.
```suggestion
        [Tooltip("Radar scope parameters")]
        public RadarParametersScope[] scopeParameters;
```
And the same statement also refers to this structure name - to be updated if you decide to follow this proposition.